### ISSUE TITLE
Hybrid approach

### ISSUE THREAD

**Issue by sanchezvvictor** [InterDigitalInc/FeatureStyleEncoder#16]
Hi,

I am using your approach to generate an initial latent and feature space for an hybrid approach.
During optimization, the loss I use for feature reconstruction is the same as the one used for training the encoder: F.mse_loss(features_in, features_out)

I am obtaining nearly perfect inversion but when I want to edit (only the latent space) I have results that are too close to the inverted. (Noting that I realized the edition of the features w.r.t formula 1 in your paper).

I am guessing that this might be because of the loss on feature reconstruction during inversion.

Do you have any suggestion for improving edition ?

Thanks
